### ISSUE TITLE
Clarifying the maintainers of the json gem

### ISSUE THREAD

**Issue by eregon** [flori/json#412]
@flori As mentioned in https://github.com/flori/json/pull/402#issuecomment-586585888,
It would be great to have other maintainers for such a critical part of the Ruby standard library.
Can we consider @hsbt @nurse and @headius as maintainers alongside you, which means they should be able to make all decisions needed to maintain this project? (including merging PRs, adding support for new implementations, publish releases, push the gem, etc).
From https://bugs.ruby-lang.org/issues/15605 and many reports before, it's clearly not enough to have a single not-so-active maintainer.
I don't think it's sensible to require the project creator to review/merge every PR.
Therefore I'd like to address the general situation here.

Maybe transferring the project to https://github.com/ruby/json would help to clarify this is a Ruby community-maintained gem. Most other gems in the standard library are under https://github.com/ruby, so I think it would be consistent.

@flori What do you think of that?

There are many [open PRs](https://github.com/flori/json/pulls) and issues with no response, I think it's high time to have more maintainers for this gem.

_Originally posted by @eregon in https://github.com/flori/json/pull/402#issuecomment-586585888_

**Comment by marcandre** [comment]
I can help too.

Where is @flori ?

**Comment by simi** [comment]
Ping me as well if help is needed.

**Comment by hsbt** [comment]
I and @nurse can merge the pull request and maintain the json-2.x releases. I'm not sure what @eregon's request. If you have a issues, you can submit the pull request, triage and more.

**Comment by marcandre** [comment]
> I and @nurse can merge the pull request and maintain the json-2.x releases. I'm not sure what @eregon's request. If you have a issues, you can submit the pull request, triage and more.

Great.

Will this repo move to `ruby/json`?

Many PRs and issues are outstanding. I will try to help with triage.

**Comment by hsbt** [comment]
> Will this repo move to `ruby/json`?

It's flori's convenience. I and @nurse won't hard-fork to ruby/json from flori/json. It's great to transfer to ruby org from this repository. I'm waiting the decision of @flori

**Comment by headius** [comment]
I already have push rights for the gem but not the repository.

**Comment by eregon** [comment]
My understanding is @hsbt, @nurse and @headius can push a new release in case of need (e.g., security issue), but otherwise they do not consider themselves maintainers.
For example, https://github.com/flori/json/pull/402#issuecomment-584580222 took forever to merge, which I believe is for this reason.

So what we need is full maintainers which help maintain the `json` gem and reduce the backlog.
So far there is a growing number of PRs on this repo, most not even reviewed.

I suggest to add @marcandre to maintainers as he's willing to help here.
Can anyone add @marcandre as a collaborator to this repo? Or only @flori?

I have nothing against @flori but it's a fact he only replies on this gem's tracker once every few months, does not reply to emails often, and ignored this issue so far, even though it's been going on [for years](https://bugs.ruby-lang.org/issues/15605).

So, please, @flori, can you let others be full maintainers alongside you, such that the json gem is well maintained?
That is critical to the Ruby ecosystem.
That means those maintainers should have all rights they need: repo push and at least for some gem push, and rights to add more maintainers. I think it would make sense to give @hsbt admin rights, so we don't need to constantly ping @flori if we need any admin-level action.

It's also clear we're all here in favor of moving the repository to the ruby organization, like most other standard libraries/default gems, would that be OK @flori? If so, can you transfer the repository to @hsbt, who will then transfer it to the Ruby organization? (that's the standard procedure on GitHub to transfer a repo from a given user)

If any of you knows a better way to contact @flori, could they ask him to look at this issue?

**Comment by eregon** [comment]
Also, can we give @headius push rights?
It's obviously very important to help maintain the [JRuby extension of this gem](https://github.com/flori/json/tree/master/java/src/json/ext).

**Comment by marcandre** [comment]
All open PRs have been reviewed; most can be merged or closed, a few are waiting on executive decisions.

There seems to be work to be done on the JRuby side and the tests suite is not currently passing. Who can work on this?

**Comment by hsbt** [comment]
I and @nurse can merge PRs and release `json` gem.  and @marcandre handles exists or new issue and PRs. There is no issues json gem now.

I will close this.

**Comment by eregon** [comment]
Great!
